### ISSUE TITLE
"Show Applications" click behavior with third-party docks/panels

### ISSUE THREAD

**Issue by grexen** [pop-os/gnome-shell-extension-pop-cosmic#346]
When I click "Show Applications" in my dock, the application overview opens. When I click again it opens again instead of closing, which would be expected.

**Comment by n3m0-22** [comment]
Are you still seeing this behavior. Do you have any 3rd party extensions installed?

**Comment by jacobgkau** [comment]
Please also provide the output of hte command `apt policy pop-cosmic gnome-shell-extension-cosmic-dock` to confirm the versions of those packages you have installed.

**Comment by grexen** [comment]
> Are you still seeing this behavior. Do you have any 3rd party extensions installed?

Yes I'm using "Dash-to-panel" and Cosmic-Dock is disabled - so it's heaviliy customized. It would be cool if it would work in that combination but can't blame the Cosmic desktop for that.

**Comment by grexen** [comment]
> Please also provide the output of hte command `apt policy pop-cosmic gnome-shell-extension-cosmic-dock` to confirm the versions of those packages you have installed.
```

pop-cosmic:
  Installiert:           0.1.0~1651251139~22.04~9c251e7
  Installationskandidat: 0.1.0~1651251139~22.04~9c251e7
  Versionstabelle:
 *** 0.1.0~1651251139~22.04~9c251e7 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
       1001 http://apt.pop-os.org/release jammy/main i386 Packages
        100 /var/lib/dpkg/status
gnome-shell-extension-cosmic-dock:
  Installiert:           72~ubuntu2pop0~1653423268~22.04~02891fe
  Installationskandidat: 72~ubuntu2pop0~1653423268~22.04~02891fe
  Versionstabelle:
 *** 72~ubuntu2pop0~1653423268~22.04~02891fe 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
       1001 http://apt.pop-os.org/release jammy/main i386 Packages
        100 /var/lib/dpkg/status

```

**Comment by jacobgkau** [comment]
> Yes I'm using "Dash-to-panel" and Cosmic-Dock is disabled - so it's heaviliy customized. It would be cool if it would work in that combination but can't blame the Cosmic desktop for that.

The issue should not be opened in the COSMIC Dock repo if you have COSMIC Dock disabled. I've moved the issue to the COSMIC repo. I believe either COSMIC or COSMIC Dock handles the Applications button specially to prevent this issue from occurring. Either the same special handling would need to be added to dash-to-panel, or COSMIC would need to not require that special handling.
